blosc/shuffle.c

```diff
@@ -312,12 +393,11 @@ static void
 unshuffle4(uint8_t* dest, const uint8_t* orig, size_t size)
 {
   size_t i, j, k;
-  size_t neblock, numof16belem;
-  __m128i xmm0[4], xmm1[4];
+  const size_t numof16belem = size / (4 * sizeof(__m128i));
 
-  neblock = size / 4;
-  numof16belem = neblock / 16;
   for (i = 0, k = 0; i < numof16belem; i++, k += 4) {
+    __m128i xmm0[4], xmm1[4];
```

Comment:
Is this an example of intermingled declarations and code that MSVC would have problems with, or is this supported?